indra/sources/crog/processor.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+"""Processor for the `Chemical Roles Graph (CRoG)
+<https://github.com/chemical-roles/chemical-roles>`_.
+"""
+
+from ..utils import RemoteProcessor
+
+__all__ = [
+    'CrogProcessor',
+]
+
+CROG_URL = 'https://raw.githubusercontent.com/chemical-roles/' \
+           'chemical-roles/master/docs/_data/crog.indra.json'
+
+
+class CrogProcessor(RemoteProcessor):
+    """A processor for the Chemical Roles Graph."""
+
+    def __init__(self):
+        super().__init__(url=CROG_URL)
```

Comment:
True, but my opinion is that Identifiers.org is wrong. The ec-code:1.5.1 is part of the hierarchy and gives us a lot of really useful information. The bioregistry validation overrides the identifiers.org pattern (see https://github.com/cthoyt/bioregistry/blob/1270930a4b199801cc2e0ca53de4939ae7c4f724/src/bioregistry/data/bioregistry.json#L6050).

Maybe we need to update the PybelProcessor since it's possible for PyBEL citations to have DOI prefixes instead of PMIDs. I'll look into that

Reply:
I made the PyBEL processor more generally able to handle different kinds of citations in d8c4d0068